fire-hose-management-frontend/src/components/Departments/DepartmentOverlay.jsx

```diff
@@ -1,158 +1,204 @@
+    }
+}));
 
 
-i18nIsoCountries.registerLocale( require( "i18n-iso-countries/langs/de.json" ) );
+i18nIsoCountries.registerLocale(require("i18n-iso-countries/langs/de.json"));
 
-const countries = Object.values( i18nIsoCountries.getNames( "de", { select: "official" } ) ).map( ( country, index ) =>
-  <MenuItem key={index} value={country}>{country}</MenuItem>
-)
+const countries = Object.values(i18nIsoCountries.getNames("de", {select: "official"})).map((country, index) =>
+    <MenuItem key={index} value={country}>{country}</MenuItem>)
 
+const genderMenuItems = [<MenuItem key={0} value={"MALE"}>Herr</MenuItem>, <MenuItem key={1} value={"FEMALE"}>Frau</MenuItem>,
+    <MenuItem key={2} value={"OTHER"}>Divers</MenuItem>]
 
-export const DepartmentOverlay = ( {
-                                     handleClose,
-                                     headline,
-                                     submitBtnText,
-                                     submitBtnFunction,
-                                     initialDepartment = "",
-                                     initialStreet = "",
-                                     initialHouseNumber = "",
-                                     initialPostalCode = "",
-                                     initialLocation = "",
-                                     initialCountry = "Deutschland",
-                                     initialForename = "",
-                                     initialSurname = "",
-                                     initialMail = ""
-                                   } ) => {
 
-  const classes = useStyles();
-  const [department, setDepartment] = useState( initialDepartment )
-  const [street, setStreet] = useState( initialStreet )
-  const [houseNumber, setHouseNumber] = useState( initialHouseNumber )
-  const [postalCode, setPostalCode] = useState( initialPostalCode )
-  const [location, setLocation] = useState( initialLocation )
-  const [country, setCountry] = useState( initialCountry )
-  const [forename, setForename] = useState( initialForename )
-  const [surname, setSurname] = useState( initialSurname )
-  const [mail, setMail] = useState( initialMail )
+export const DepartmentOverlay = ({
+                                      handleClose,
+                                      headline,
+                                      submitBtnText,
+                                      submitBtnFunction,
+                                      initialDepartmentName = "",
+                                      initialStreet = "",
+                                      initialHouseNumber = "",
+                                      initialPostalCode = "",
+                                      initialLocation = "",
+                                      initialCountry = "Deutschland",
+                                      initialGender = "",
+                                      initialFirstname = "",
+                                      initialLastname = "",
```

Comment:
Codacy has a fix for the issue: [Delete `····································`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
  initialLastname = "",
```